family2family/src/main/java/org/cbtf/f2f/util/LoggedInUtil.java

```diff
@@ -0,0 +1,19 @@
+package org.cbtf.f2f.util;
+
+import org.springframework.security.authentication.AnonymousAuthenticationToken;
+import org.springframework.security.core.context.SecurityContextHolder;
+
+/**
+ * Helper utility to determine if the user is or is not logged in.
+ */
+public class LoggedInUtil {
```

Comment:
Not absolutely necessary to do. Since our spring security forwards to google by default, if you hit a secure page it will just auto redirect you to google to log in.